src/cart_total/cart_total.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import './cart_total.css'
+import { useDispatch, useSelector } from 'react-redux';
+import { changeNumberProduct, deleteProduct, useTotalCost } from '../features/todo/todoSlice';
+import { Link } from 'react-router-dom';
+
+const costChecker = (cost, number) => {
+    const costString = cost;
```

Comment:
why do you need to create new variable `costString`? it already comes in your props as `cost`